preprocessing.py

```diff
@@ -1 +1,34 @@
 #This is for preprocessing: First will be relabeling everything and then will be splitting the data
+
+# Freya will do relabeling here:
+
+
+# Data splitting
+import torch, sklearn
+import data_reader
+import os
+import numpy as np
+
+# Open train.csv
+reader = data_reader.data_reader()
+csv_path = os.path.join(os.path.curdir, "dataset", "train.csv")
+reader.open_csv(csv_path)
+
+#Separate labels and features
+i = 1
+data = []
+data.append(reader.read(3000000))
```

Comment:
Size of data changes as we e.g. discard neutral reviews, so no hard coded values in the function